static/downloadables/runPrsCLI.sh

```diff
@@ -396,16 +429,20 @@ correctly formatted for calculations to occur. "
 include a header line with named columns. The required columns are: ${MYSTERYCOLOR}Study ID${NC}, \
 ${MYSTERYCOLOR}Trait${NC}, ${MYSTERYCOLOR}RsID${NC}, ${MYSTERYCOLOR}Chromosome${NC}, ${MYSTERYCOLOR}Position${NC}, \
 ${MYSTERYCOLOR}Risk Allele${NC}, ${MYSTERYCOLOR}Odds Ratio${NC}, and ${MYSTERYCOLOR}P-value${NC}. \
-Optional column headers that will be included if present are: ${MYSTERYCOLOR}Citation${NC} and \
-${MYSTERYCOLOR}Reported Trait${NC}. Column order does not matter and there may be extra columns \
-present in the file. Required and optional header names must be exact."
-    echo ""
-    echo "If more than one odds ratio exists for an RsID in a study/trait combination, the snp will be excluded."
+If the ${GREEN}-b${NC} flag is present, then instead of an ${MYSTERYCOLOR}Odds Ratio${NC} column, \
+the user should include a ${MYSTERYCOLOR}Beta Coefficient${NC} column and a ${MYSTERYCOLOR}Beta Unit${NC} column. \
+Optional column headers that will be included if present are: ${MYSTERYCOLOR}Citation${NC}, ${MYSTERYCOLOR}P-value Annotation${NC}, \
+${MYSTERYCOLOR}Beta Annotation${NC}, and ${MYSTERYCOLOR}Reported Trait${NC}. Column order does not matter and there may be extra columns \
+present in the file. Required and optional header names must be exact. Note that if P-value Annotation and/or Beta Annotation \
+are present, then the calculator will separate calculations by those columns. If you do not wish for this to \
+happen, do not include those optional columns."
+    # echo ""
```

Comment:
Can these commented out echos be removed?

Reply:
updated the wording and put the echos back in.